salesforce/10.10/modules/ROOT/pages/salesforce-connector-processing-events.adoc

```diff
@@ -65,6 +65,12 @@ If the cluster is stopped, the data from the object store is lost, so when the c
 If you need data to be persisted so that the application can continue processing events when restarted, the object store can be configured to use a database as the persistence layer. For more information, see xref:mule-runtime::creating-and-managing-a-cluster-manually.adoc[Creating and Managing a Cluster]).
 * When running the application in CloudHub, if you are using multiple workers, all of the workers receive and process the events.
 
+[NOTE]
+---
+It is best that you do not have two or more flows using a source that listens to events on the same channel or topic because the events will be consumed twice from your user quota. If you have a use case that requires processing the same events in two or more different flows, consider implementing that logic in a single flow instead.
+
+Having multiple sources listening to the same events and having the *Resume form Last Replay ID* option enabled leads to data corruption and potential loss of events.
```

Comment:
typo: form > from  
The field name is actually *Resume from the Last Replay Id*